zebra-network/Cargo.toml

```diff
@@ -9,7 +9,8 @@ edition = "2021"
 
 [features]
 default = []
-tor = ["arti-client", "tor-rtcompat"]
+# Wait until `arti-client`'s dependency `x25519-dalek v1.2.0` is updated to a higher version.
```

Comment:
This looks like a good change, and I'm happy to get rid of the dependency on an outdated version of `arti-client`.

Can you open a ticket explaining that `arti-client` needs to upgrade to `x25519-dalek v2` to resolve a conflicting `zeroize` dependency?

Then we can reference that ticket from all the tor changes, rather than explaining it every time.

Reply:
We might also want to mention that ticket in the `tor` feature docs:
https://github.com/ZcashFoundation/zebra/blob/5a6c84024c759048116733881fe2b0c5c890d65a/zebra-network/src/lib.rs#L60-L61

And in the known issues in the README.

Reply:
I added the references to the ticket, but I kept the explanations since they were one-liners. I thought it would be faster to read that line instead of visiting the ticket itself to peek at what's going on. If that doesn't suit, I'll remove the explanations.